### PR TITLE
Open reddit link

### PR DESCRIPTION
**Pull request by LucyWilcox**

### CODE REVIEW

File: app/src/main/java/com/mobileproto/hireddit/hireddit/reddit/RedditSearcher.java [hunk 1]
```diff
@@ -128,7 +130,9 @@ public void filterComment(ArrayList<String> allComments) {
         }
     }
 
-    public String pickComment(ArrayList<String> allComments) {
+    public String pickComment(Hashtable<String, ArrayList<String>> allCommentsHash) {
```
Comment (nmohamed) [inline app/src/main/java/com/mobileproto/hireddit/hireddit/reddit/RedditSearcher.java:132]: Extra > in there?

Comment (nmohamed) [inline app/src/main/java/com/mobileproto/hireddit/hireddit/reddit/RedditSearcher.java:132]: Oh lol nvm

**Comment by LucyWilcox** [comment]
Hey guys, this branch seems buggy. But I'm not sure what's up with it. Lets not merge it for now though.

Update: seems to be fixed now

File: app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeechFragment.java [hunk 2]
```diff
@@ -186,17 +196,17 @@ public void errorCallback(int errorCode, int numErrors) {
     }
 
     @Override
-    public void commentCallback(String comment) {
+    public void commentCallback(String comment, ArrayList<String> linkInfo) {
         if (comment == null) {
             Log.d(DEBUG_TAG, "No valid comments found");
             Toast.makeText(getContext(), "No valid comments available", Toast.LENGTH_SHORT).show();
         } else {
+            link = "https://www.reddit.com/comments/" + linkInfo.get(0) + "/_/" + linkInfo.get(1) + "?context=2";
```
Comment (davidabrahams) [inline app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeechFragment.java:216]: comment why context=2.

Also, we might want to later create this URL using a Builder, but that's not that important as long as this works for all cases.

**Pull request merged**